.github/workflows/pr.yml

```diff
@@ -554,13 +555,25 @@ jobs:
         id: test_notebook_vscode
         if: matrix.test-suite == 'notebook'
 
+      - name: Run Remote with VSCode & Jupyter
+        uses: GabrielBB/xvfb-action@v1.4
+        with:
+          run: npm run testRemoteNativeNotebooksInVSCode
```

Comment:
Start real jupyter and connect to that.
All `remote` tests will connect & use the same jupyter server. Not going to start & stop for every test suite.